documentation/topics/relationships.md

```diff
@@ -1,101 +1,73 @@
 # Relationships
 
-Relationships are a core component of Ash. They provide a mechanism to describe the relationships between your resources, and through those relationships you can do things like
+Relationships describe the connections between resources and are a core component of Ash. Defining relationships enables you to do things like
 
 - Loading related data
 - Filtering on related data
 - Managing related records through changes on a single resource
 - Authorizing based on the state of related data
 
-## Customizing default belongs_to attribute type
+## Relationships Basics
 
-By default, we assume foreign keys that we add by default (for `belongs_to` relationships) should be `:uuid`. To change this default, set the following configuration:
+A relationship must exist between a source resource and a destination resource. These are defined in the `relationships` block of the source resource. For example, if `MyApp.Tweet` is the source resource, and `MyApp.User` is the destination resource, we could define a relationship called `:owner` like this:
```

Comment:
the word `must` here might imply that you *must* have a relationship? I see what you are saying though.